lua/dap/repl.lua

```diff
@@ -75,6 +77,10 @@ function M.execute(text)
     session:_step('stepIn')
   elseif vim.tbl_contains(M.commands.out, text) then
     session:_step('stepOut')
+  elseif vim.tbl_contains(M.commands.up, text) then
+    session:_frame_delta(1)
```

Comment:
Do you think it would make sense to print the frames within the range of the current frame - let's say 2 automatically afterwards? 

So you'd get:

```
dap> .up

  a()
  b()
→ c()
  d()
  e()
```

Reply:
:+1: